colossalai/tensor/compute_spec.py

```diff
@@ -22,4 +22,7 @@ def __init__(self, compute_pattern: ComputePattern) -> None:
         self.output_replicate = True
 
     def __repr__(self):
-        return f'compute pattern: {self.compute_pattern}'
+        return f'Compute pattern: {self.compute_pattern}'
+
+    def set_keeping_output(self):
```

Comment:
Not used!
signature had better to be set_output_replicate(self, flag : bool)

Reply:
It will be used in an upcoming unit test right away.